definitions/npm/lodash_v4.x.x/flow_v0.63.x-/lodash_v4.x.x.js

```diff
@@ -1619,7 +1619,7 @@ declare module "lodash/fp" {
     // Array
     chunk<T>(size: number): (array: Array<T>) => Array<Array<T>>;
     chunk<T>(size: number, array: Array<T>): Array<Array<T>>;
-    compact<T, N: T>(array: Array<N>): Array<T>;
+    compact<T, N: ?T>(array?: ?$ReadOnlyArray<N>): Array<T>;
```

Comment:
Is `compact` callable without passing in an array?

Reply:
Virtually everything in `lodash/fp` can be called without providing the "expected" collection, and it seems like the definitions overlook this in general, a concern that I recently brought up [here](https://github.com/flowtype/flow-typed/pull/1262#issuecomment-382199299). In the case of `compact`, calling with no value or a nil value will return an empty array (which trivially qualifies as an `Array<T>`).